### PR TITLE
Market data tutorials

### PR DESCRIPTION
**Pull request by rizwansaeed**

### CODE REVIEW

File: sdk/tests/tutorials/marketdata/test_instruments.py [hunk 1]
```diff
@@ -3,17 +3,44 @@
 import lusid
 import lusid.models as models
 from lusid.utilities.api_client_builder import ApiClientBuilder
+from lusid.exceptions import ApiException
 from utilities.credentials_source import CredentialsSource
+from utilities.test_data_utilities import TestDataUtilities
 
 
 class Instruments(unittest.TestCase):
+    property_definitions_api = None
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/marketdata/test_instruments.py]: Why do you set this as None here?

File: sdk/tests/tutorials/marketdata/test_quotes.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.quotes_api = lusid.QuotesApi(api_client)
+
+    def test_add_quote(self):
+
+        request = models.UpsertQuoteRequest(
+            quote_id=models.QuoteId(
+                provider="DataScope",
+                price_source="USDRC",
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/marketdata/test_quotes.py]: Change all price_source to be "" as this is the new default.

Comment (rizwansaeed) [reply sdk/tests/tutorials/marketdata/test_quotes.py]: Remove `price_source`

File: sdk/tests/tutorials/marketdata/test_instruments.py (hunk 1 above)
Comment (rizwansaeed) [reply sdk/tests/tutorials/marketdata/test_instruments.py]: Removes an editor warning

**Pull request merged**